src/main/scala/codacy/metrics/ScalaParser.scala

```diff
@@ -0,0 +1,123 @@
+package codacy.metrics
```

Comment:
Yes. Deleted a lot of stuff and condensed the classes and methods count into a single "countClassesAndMethods" function, instead of traversing the ast twice.

Which means I edited the companion object, but the class stays the same (with a new name).